jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -765,7 +765,7 @@ private String getUserDefinedData() {
         try {
             return data != null ? data.getData() : null;
         } catch (ContextNotActiveException e) {
-            LRALogger.i18nLogger.warn_missingContexts("CDI bean of type LRAParticipantData is not available", e);
+            LRALogger.i18nLogger.warn_missingContexts("LRAParticipantData is not usable in this (probably async context.");
```

Comment:
missing ')' , I would probably remove the "(probably async)" part

Reply:
And is it definitely intentional to remove the cause?

Reply:
The cause isn't giving much more context and it's just a stacktrace in the log. Since we can't get an instance in async cases, this is essentially unsupported and IMO then should log a stacktrace.

Reply:
We do have a long standing informal approach which is to kind of include the log messages as part of the API, but I think it _might_ be OK for LRA if @marcosgopen does but I do think you should track the change in message structure in some way it could be identified from "release notes" (maybe someone has pattern matching on a log message) should such a thing be what Marco would like for the LRA project.

Reply:
Also, would you be able to comment on the "(probably async)" comment from Marco, please?

Reply:
Thanks @tomjenkinson for you comment. I agree the log message change is something we want to track down in the release notes. @xstefank could I ask you to create a JBTM for this PR and add a release note there about the message change please? Please also change the commit message and PR title with a reference to that JBTM.  Then I would be glad to approve your PR.